src-electron/generator/helper-session.js

```diff
@@ -294,7 +294,8 @@ async function all_user_cluster_attribute_util(
       await queryAttribute.selectAllAttributeDetailsFromEnabledClusters(
         currentContext.global.db,
         endpointsAndClusters,
-        packageIds
+        packageIds,
+        side
```

Comment:
This used to not restrict to a particular side, right?  Why is this change OK?

Reply:
It defaults to null when not specified and the query is handled accordingly in that case.